src/Header.css

```diff
@@ -0,0 +1,45 @@
+.input-card{
+    border: 1px solid black;
+    margin-top: 3%;
+    box-shadow: 3px 3px 15px 1px rgb(199 199 199);
+}
```

Comment:
implement tailwind in your app and Remove any un-necessary styling from your code. @Kunal-inno